src/library/features/crates/cratefeaturehelper.cpp

```diff
@@ -48,28 +48,27 @@ CrateId CrateFeatureHelper::createEmptyCrate() {
         if (!ok) {
             return CrateId();
         }
+        newCrate.setName(newCrate.getName().simplified());
         if (!newCrate.hasName()) {
             QMessageBox::warning(
-                    nullptr,
-                    tr("Creating Crate Failed"),
-                    tr("A crate cannot have a blank name."));
+              nullptr,
```

Comment:
This change is just a wrong reformatting. Please adjust your indentation settings with the Mixxx coding guidelines: "Use double indent (8-spaces) for broken lines"